src/website/views/custom-elements/button.pug

```diff
@@ -36,57 +36,57 @@ p.-text
   :code(lang='html')
     <!-- For light backgrounds -->
     <chi-button>Base</chi-button>
-    <chi-button color='primary'>Primary</chi-button>
-    <chi-button color='dark'>Dark</chi-button>
+    <chi-button color="primary">Primary</chi-button>
+    <chi-button color="dark">Dark</chi-button>
 
     <!-- For dark backgrounds -->
-    <chi-button color='secondary'>Secondary</chi-button>
-    <chi-button color='light'>Light</chi-button>
+    <chi-button color="secondary">Secondary</chi-button>
+    <chi-button color="light">Light</chi-button>
 
 
 h4 Outline
 p.-text
-  | To remove a buttons solid background and keep it's colored border, apply the attribute
-  | <code>outline</code> on the <code>&lt;chi-button&gt;</code> tag.
+  | To remove a buttons solid background and keep it's colored border, set the attribute
+  | <code>variant</code> value to <code>outline</code>.
 .example.-mb--3
   .a-grid.-noGutter
     .a-col.-w--12.-w-md--6
       .-p--3.-d--flex(style="justify-content: center;")
         each type, val in {Primary:'primary', Dark:'dark'}
           div.-pr--2
-            chi-button(color=type, outline)= val
+            chi-button(color=type, variant="outline")= val
```

Comment:
@srinath-koneru js attributes should be followed by single quotes and html attributes should have double quotes in order to follow best practices. Please change it in these files.